dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -327,6 +327,17 @@
 			interrupts = <53 0>;
 			label = "TRNG";
 		};
+
+		usbd1: usbd@402E0000 {
+			compatible = "nxp,kinetis-usbd";
+			reg = <0x402E0200 0x1000>;
+			interrupts = <113 1>;
+			interrupt-names = "usb_otg";
+			num-bidir-endpoints = <8>;
+			maximum-speed = "full-speed";
```

Comment:
shouldn't this be high-speed?

Reply:
The Zephyr USB stack is not fully HS compatible, for example bulk endpoints do not have the correct size in HS speed. It's therefore a good idea to limit the speed to FS by default, an app fully validated on HS can override the default.